src/panels/config/labels/dialog-label-detail.ts

```diff
@@ -118,6 +122,14 @@ class DialogLabelDetail
                 "ui.panel.config.labels.detail.color"
               )}
             ></ha-color-picker>
+            <ha-textfield
```

Comment:
I think we should make this a textarea instead of a field?

Reply:
Makes sense. Updated PR to use textarea for the description